DataScience/IntroToPython/IntroToPythonPandas/blog.md

```diff
@@ -82,15 +82,13 @@ We have our data with us, but it’s very large, so we might be confused as to w
 |:-----:|------:|:-----:|------:|
 |data|DataFrame|(10,15)|Column names: sl_no, gender, ssc_p, ssc_b, hsc_p, hsc_b, hsc_s, degree...|
 
-## Exercise 1
+## Getting Top and Bottom 6 Salaries
 
 On the Kaggle page used to download our csv, there is a part of the page providing definitions to our data’s column headers, including salary. Use it, and the newly learned commands to:
 Get the 6 highest and lowest salary entries from our dataset.
 
 Tip: calling data.head() returns a text answer, but for a new dataframe, write new_data = data.head()
 
-## Solution
-
 ```py
 salary = data.sort_values(["salary"])
```

Comment:
Only a suggestion, but what data are you dealing with exactly here? The code does mention salaries but adding more context would be great here.